app/Services/ExchangeRateGetter.php

```diff
@@ -4,68 +4,93 @@
 class ExchangeRateGetter {
 	public function get() {
```

Comment:
`get` method have to take parameter `date` which by default is null
```suggestion
	public function get(?string $date = null) {
```